src/v/pandaproxy/json/iobuf.h

```diff
@@ -102,4 +102,35 @@ class rjson_serialize_impl<iobuf> {
     serialization_format _fmt;
 };
 
+///\brief Wrap a std::istream around an iobuf
```

Comment:
nit: should these go into somewhere in `bytes/` instead of `json/` to make them easier to reuse? They are similar to `iobuf_istreambuf` in `streambuf.h`

Reply:
> nit: should these go into somewhere in bytes/ instead of json/ to make them easier to reuse? They are similar to iobuf_istreambuf in streambuf.h

yeh, looks like it. bytes/iostream bytes/streambuf.h etc... seem like good spots

Reply:
Done